cmake/modules/FindUFE.cmake

```diff
@@ -50,6 +50,10 @@ if(UFE_INCLUDE_DIR AND EXISTS "${UFE_INCLUDE_DIR}/ufe/ufe.h")
 
     if("${UFE_MAJOR_VERSION}" STREQUAL "0")
         math(EXPR UFE_PREVIEW_VERSION_NUM "${UFE_MINOR_VERSION} * 1000 + ${UFE_PATCH_LEVEL}")
+    elseif("${UFE_VERSION}" STREQUAL "4.0.0")
+        # Temporary. Once next Maya PR is released with UFE v4.0.0 this should
+        # be removed (along with all the UFE_PREVIEW_VERSION_NUM checks).
+        set(UFE_PREVIEW_VERSION_NUM 4045)
```

Comment:
See [PR 1926](https://github.com/Autodesk/maya-usd/pull/1926) where I did the same thing for the transition to Ufe v3.